transformer/compose/v1v2.go

```diff
@@ -124,6 +124,18 @@ func parseV2(path string, interpolate bool) (*project.Project, error) {
 	return proj, nil
 }
 
+// parseV2 parses version 2 compose files while capturing panics
+func parseV2(path string, interpolate bool) (result *project.Project, err error) {
+	defer func() {
+		if r := recover(); r != nil {
+			logrus.Errorf("recovered from panic in panicky_parseV2: %q", r)
+			err = fmt.Errorf("panicky_parseV2 failed: %q", r)
+		}
+	}()
+	result, err = panicky_parseV2(path, interpolate)
```

Comment:
isn't it sufficient to wrap only `err = proj.Parse()` ?

Reply:
done